vignettes/hubverse_ms_vignette.Rmd

```diff
@@ -0,0 +1,331 @@
+
+```{r, fig.cap = "Forecasts of hospital admissions in Texas due to influenza in the 2024-2025 respiratory virus season. Predictions from five submitted models and an ensemble are shown along with observed data. The observed data are shown as a line with dots for specific weekly observations. Each individual model is shown in grey and the mean ensemble is shown in blue, with the 80% prediction intervals shown as shaded regions."}
+ensemble_forecast <- hubEnsembles::simple_ensemble(flusight_data)
+
+all_forecasts <- flusight_data |>
+  bind_rows(ensemble_forecast)
+```
+
+We can now visualize the ensemble forecast as well as individual model performance all together. 
+
+```{r}
+all_forecasts |>
+  hubVis::plot_step_ahead_model_output(target_data,
+    x_col_name = "target_end_date",
+    x_target_col_name = "target_end_date",
+    use_median_as_point = TRUE,
+    group = "reference_date",
+    interactive = FALSE,
+    fill_transparency = .2,
+    intervals = 0.8,
+    one_color = "darkgray",
+    ens_name = "hub-ensemble",
+    ens_color = "blue",
+    pal_color = NULL
+  ) +
+  theme(legend.position = "none") +
+  ylab("hospital admissions due to influenza")
+```
+
+\clearpage
+
+# Evaluate forecasts
+
+In many settings, point and probabilistic forecasts can be compared with eventual observations to provide a quantitative evaluation of model performance. These evaluations can provide insights into the comparative accuracy of different forecasting approaches, and can help identify areas for improvement in the models.
+
+The package `hubEvals` can be used to evaluate probabilistic predictions against observed data. The `hubEvals` package creates an interface for hubverse-style model output and oracle output target data to the `scoringutils` package, which is an engine for computing a number of common scores for probabilistic predictions[@bosse-scoringutils-2024].
+
+As mentioned earlier, to evaluate model outputs we use oracle output data, a form of target data which matches and can be merged with hubverse-style model output data but which contains a point prediction representing the eventual observation, as if observed by an oracle model. 
+
+The FluSight hub also stores oracle output data in the cloud, and we can access it using the `hubData::connect_target_oracle_output()` function. This function connects to the oracle output data in the hub and returns a connection object that can again be used to query the data.
+
+```{r}
+oo_data_con <- hubData::connect_target_oracle_output(hub_path)
+```
+
```

Comment:
Note that the oracle-output contains a `horizon` column, which [as discussed](https://github.com/reichlab/decisions/pull/31#discussion_r2175444614), probably shouldn't be there?

I think we should resolve this or at least open an issue about resolving it soon in. I don't feel we should be showcasing wrongly formatted hubs in the vignette as it can cause unnecessary confusion.